terraform/service_retry.tf

```diff
@@ -110,8 +110,8 @@ module "retry_cloud_run" {
     "PROJECT_ID" : data.google_project.default.project_id,
   }
   secret_envvars = {
-    "GITHUB_SSH_KEY" : {
-      name : "github-ssh-key",
+    "GITHUB_PRIVATE_KEY" : {
```

Comment:
@verbanicm i changed this for consistency with minter but will require an infra change. Should I revert this name change?

Reply:
Well need to do the whole infra PR dance with the aggregator PR branch name......